modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/DartDioClientCodegen.java

```diff
@@ -734,7 +734,9 @@ private Set<String> rewriteImports(Set<String> originalImports, boolean isModel)
                 resultImports.add(i);
             } else if (importMapping().containsKey(modelImport)) {
                 resultImports.add(importMapping().get(modelImport));
-            } else if (modelImport.startsWith("package:")) {
+            } else if ("dart:core".equals(modelImport)) { // import dart:core directly
+                resultImports.add(modelImport);
```

Comment:
`dart:core` isn't the only one, you can also get dart:convert, dart:typed_data, dart:async, etc...

Reply:
Is there a complete list that I can include?

Reply:
well in dart you only get 3 types of imports:

1. `package:`
2. `dart:`
3. relative

Reply:
ok, i'll do a check to see if `modelImport` starts with `dart:`

Reply:
done viia https://github.com/OpenAPITools/openapi-generator/pull/14984/commits/7027fa9f39b1de90406ff3bd103fce121c73efcd